public_html/join.php

```diff
@@ -24,20 +24,20 @@
 <p>We use a few different tools to organise the nf-core community -
     you are welcome to join us at any or all!</p>
 <p class="text-center">
-  <a href="https://nf-co.re/join/slack" class="mb-2 btn btn-lg btn-success">
-    <i class="fab fa-slack"></i> Join nf-core Slack
+  <a href="https://nf-co.re/join/slack" class="mb-2 me-3 btn btn-lg btn-success">
```

Comment:
because we use the brand colors for the rest:
```suggestion
  <a href="https://nf-co.re/join/slack" class="mb-2 me-3 btn btn-lg btn-success" style="background-color: #4a154b;">
```

Reply:
Ok, but has a green border so needs to be btn-dark as well.
```suggestion
  <a href="https://nf-co.re/join/slack" class="mb-2 me-3 btn btn-lg btn-dark" style="background-color: #4a154b;">
```